### DISCUSSION TITLE
cannot run mediamtx on RPi4 or compute module

### DISCUSSION THREAD

**Discussion by sravich1** [bluenviron/mediamtx#2915] [Questions]
### Question

Hi,
when I run the mediamtx after compiling I get following error
1. not able to see the h264 using VLC
2. invalid path name ...no one is publishing in h264
2024/01/17 23:38:53 INF [RTSP] [conn 192.168.29.82:49870] opened
2024/01/17 23:38:53 INF [RTSP] [conn 192.168.29.82:49870] closed: invalid path name: can't end with a slash (h264/)
2024/01/17 23:38:57 INF [RTSP] [conn 192.168.29.82:49873] opened
2024/01/17 23:38:57 INF [RTSP] [session 3aa16f27] created by 192.168.29.82:49873
2024/01/17 23:38:57 INF [RTSP] [conn 192.168.29.82:49873] closed: no one is publishing to path 'h264'
2024/01/17 23:38:57 INF [RTSP] [session 3aa16f27] destroyed: not in use
2024/01/17 23:38:57 INF [RTSP] [conn 192.168.29.82:49874] opened
2024/01/17 23:38:57 INF [RTSP] [conn 192.168.29.82:49874] closed: EOF

My IP is different and where as the conn RPI is trying to establish is different 
where is the issue..let me know
Thanks and regards
Ravi

**Comment by aler9** [comment]
If you want support, please describe in a very detailed way how to replicate the issue and attach configuration and full logs, as written in the instruction. Otherwise it's impossible to understand what the problem is.
